### ISSUE TITLE
Media files not transfered to multisite

### ISSUE THREAD

**Issue by stuartasutton** [ld4pe/ld4pe-general#11]
In bringing up the multisite, the only image brought over for LD4PE was the LD4PE logo. Is there a way to import the remaining image file...pretty please :-)? I can reconstruct (I think), but it would take a lot of doing.

**Comment by stuartasutton** [comment]
David, do I need to manually copy imgage files from the old explore implementation to the new?

**Comment by davidwtalley** [comment]
I thought I had copied all the old files to the right directories under the new multisite install, but I see a lot of broken image links on the site. I am unable to get Filezilla to connect to double-check that or upload the files. I do have them backed up on my PC.
  
 @Joseph: Is it possible the key file you sent a while back is expired? I don't get any error message -- the connection just times out.
  
 - David
  
  
  

----------------------------------------

**Comment by josephrchapman** [comment]
It looks as though the files are online and in the correct directory but on the wordpress website it is trying to access the old, "wordpress" directory and not the new directory, "wp".
For example:  
  NetworkError: 404 Not Found - http://explore.dublincore.net/wordpress/wp-content/uploads/2015/05/dcmi_logo.png 
  but this URL has the file and it works:
  http://explore.dublincore.net/wp/wp-content/uploads/2015/05/dcmi_logo.png
  
In terms of FileZIlla, has your IP address changed?

**Comment by stuartasutton** [comment]
Joseph, I don't see how that is possible since the media folder has only the DCMI logo and four test slider images I uploaded. Most likely the image references in the documents are wrong, but since the images are not in the media folder, they can't be 'included'...

**Comment by stuartasutton** [comment]
Also, if I log into the old site (http://explore.dublincore.net/wordpress/wp-admin/), many of the images in the media folder seem present but trashed.

**Comment by davidwtalley** [comment]
The images do appear to be in place under /wp sufolders but no longer under /wordpress. Should be a pretty easy page content edit, since we only have a few pages with images so far.
  
 Re: My IP address, a change is entirely possible, as I don't pay CenturyLink for a static IP. Sigh. I will look into that and get you the updated value.
  
 - David
  
  
  

----------------------------------------

**Comment by stuartasutton** [comment]
Guys, the fact is that when logged into the new virtual LD4PE site, you
could not edit a document and simply add media because the folder where it
looks was empty. On the dashboard, when you clicked Media, there was just
the LD4PE logo and nothing else. In editing, that was the only image
available to the editor. It was not a big deal for me to reload the images
needed by the briefing papers etc., so I went ahead and loaded them so
they'd be there when editing and inserting images. I then updated the
documents to reference the newly uploaded images.
